### PR TITLE
Script that helps to setup monitoring stack

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
The script that helps to setup monitoring stack

### Screenshot/screencast of this PR
n/a

### What issues does this PR fix or reference?
metrics development environment for https://issues.redhat.com/browse/SVPI-102

### How to test this PR?
- deploy SPI on minikube
- deploy monitoring stack
- open grafana. 3 dashboards should be available: Prometheus, controller runtime, go metrics.

### CODE REVIEW

**Comment by sparkoo** [comment]
I got 
```Align time on podman VM
Error: vm podman-machine-default not found: podman-machine-default: VM does not exist```

**Comment by sparkoo** [comment]
prometheus and grafana seems deployed fine. I can see 3 dashboards and it has some data. I'm just seeing error message ^^ because I don't use podman as minikube driver.

File: hack/deploy-minikube-monitoring.sh [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+        --force-conflicts=true \
+        --server-side
+echo
+echo -n "Waiting deployment/prometheus-operator  become available: "
+kubectl wait --for=condition=Available=True deployment/prometheus-operator -n default  --timeout=30s
+
+echo
+echo "Preparing Service account"
+cat <<EOF | kubectl apply -n default -f -
+apiVersion: v1
```
Comment (sparkoo) [inline hack/deploy-minikube-monitoring.sh:22]: I'm not big fan of having all yamls in script like this

**Review by sparkoo — APPROVED** [review]
I'm ok merging as is for now

**Review by metlos — APPROVED** [review]
I have the same reservations as @sparkoo but came to the same conclusion. Let's merge this as a first step and we can iterate later if needed.

**Pull request merged**